src/test/java/bio/terra/service/snapshotbuilder/query/FieldPointerTest.java

```diff
@@ -3,39 +3,35 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 
-import bio.terra.common.CloudPlatformWrapper;
 import bio.terra.common.category.Unit;
-import bio.terra.model.CloudPlatform;
 import java.util.ArrayList;
 import java.util.List;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.ArgumentsSource;
 
 @Tag(Unit.TAG)
 class FieldPointerTest {
 
   @ParameterizedTest
-  @EnumSource(CloudPlatform.class)
-  void buildVariable(CloudPlatform platform) {
+  @ArgumentsSource(QueryTestUtils.Contexts.class)
```

Comment:
Why this change instead of creating context here?

Reply:
Since most tests that test all platforms need to create a render context, I thought it would be easier to create an argument generator that creates all contexts. It ends up making each test slightly smaller, and it should be easier to modify if we change the render context in the future.